packages/yoga/src/Avatar/native/Circle.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { UserFilled } from '@gympass/yoga-icons';
+import { func, string, number } from 'prop-types';
+import Avatar from './Avatar';
+
+const Circle = ({ src, fill, placeholder, width, height }) => {
+  const icon = placeholder || UserFilled;
```

Comment:
Could `UserFilled ` be used as the default value of `placeholder`?

```js
const Circle = ({ src, fill, placeholder = UserFilled, width, height }) => {
```

OR

```js
Circle.defaultProps = {
  placeholder: UserFilled,
};
```

Reply:
@leonardokl Done!